Public/Src/Sandbox/Windows/DetoursServices/DetouredFunctions.cpp

```diff
@@ -1778,6 +1778,224 @@ NTSTATUS NTAPI Detoured_ZwSetInformationFile(
         FileInformationClass);
 }
 
+/// Runs an injected AnyBuild shim instead of a child process, passing the
+/// original command and arguments to the shim along with, implicitly,

```

Comment:
Maybe put these functions in a seperate file?

Reply:
Added new SubstituteProcessExecution.cpp, .h